lexicons/com/atproto/admin/defs.json

```diff
@@ -127,144 +167,221 @@
             "#recordViewNotFound"
           ]
         },
-        "reportedBy": {"type": "string", "format": "did"},
-        "createdAt": {"type": "string", "format": "datetime"},
-        "resolvedByActions": {"type": "array", "items": {"type": "ref", "ref": "com.atproto.admin.defs#actionView"}}
+        "reportedBy": { "type": "string", "format": "did" },
+        "createdAt": { "type": "string", "format": "datetime" },
+        "resolvedByActions": {
+          "type": "array",
+          "items": { "type": "ref", "ref": "com.atproto.admin.defs#actionView" }
+        }
       }
     },
     "repoView": {
       "type": "object",
-      "required": ["did", "handle", "relatedRecords", "indexedAt", "moderation"],
+      "required": [
+        "did",
+        "handle",
+        "relatedRecords",
+        "indexedAt",
+        "moderation"
+      ],
       "properties": {
-        "did": {"type": "string", "format": "did"},
-        "handle": {"type": "string", "format": "handle"},
-        "email": {"type": "string"},
-        "relatedRecords": {"type": "array", "items": {"type": "unknown"}},
-        "indexedAt": {"type": "string", "format": "datetime"},
-        "moderation": {"type": "ref", "ref": "#moderation"},
-        "invitedBy": {"type": "ref", "ref": "com.atproto.server.defs#inviteCode"},
-        "invitesDisabled": {"type": "boolean"}
+        "did": { "type": "string", "format": "did" },
+        "handle": { "type": "string", "format": "handle" },
+        "email": { "type": "string" },
+        "relatedRecords": { "type": "array", "items": { "type": "unknown" } },
+        "indexedAt": { "type": "string", "format": "datetime" },
+        "moderation": { "type": "ref", "ref": "#moderation" },
+        "invitedBy": {
+          "type": "ref",
+          "ref": "com.atproto.server.defs#inviteCode"
+        },
+        "invitesDisabled": { "type": "boolean" }
+      }
+    },
+    "subjectRepoView": {
+      "type": "object",
+      "nullable": [
+        "creator",
+        "handle",
+        "indexedAt",
+        "displayName",
+        "cid",
+        "banner",
+        "avatar",
+        "uri",
+        "description"
+      ],
+      "properties": {
+        "creator": { "type": "string", "format": "did" },
+        "handle": { "type": "string", "format": "handle" },
+        "displayName": { "type": "string" },
+        "description": { "type": "string" },
+        "banner": { "type": "string" },
+        "avatar": { "type": "string" },
+        "uri": { "type": "string" },
+        "cid": { "type": "string", "format": "cid" },
+        "indexedAt": { "type": "string", "format": "datetime" }
       }
```

Comment:
This looks like it is designed to be a view of the `app.bsky.actor.profile` record.  We might want to rethink that a little bit, as we're trying to keep the `com.atproto.*` namespace agnostic to what's happening in `app.bsky*`.

As a way to deal with this, on `#repoView` you'll see there's a `relatedRecords` property which we use to provide miscellaneous app-specific records that might provide context to the repository, including `app.bsky.actor.profile`.  Would that do the trick for us?  If so, perhaps we'd actually be just as well off using a `#repoView` rather than introducing a new kind of repo view.  What do you think?

Reply:
After rethinking this a bit, I think I was trying to pack too much into this when in reality, we really just need the `handle` for display purposes. avatar was the only other thing I was using from here on the UI at least. So, I think if we just let go of profile entirely and only hydrate this with the handle, that solves the problem of mixing bsky stuff in here along with reducing the scope of this while achieving the primary goal. wdyt @devinivy ?

Reply:
That sounds pretty good!  I would also recommend keeping the `did` around, since that's the primary identifier.  Would there be any downsides to using `#repoView` here?  I think there's an upside to avoid having many subtly different representations of a repo (but only if it actually makes sense for us, of course).